src/axom/sidre/examples/sidre_generateindex.cpp

```diff
@@ -166,7 +199,7 @@ void setup_blueprint_fields(DataStore* ds, Group* fields)
   Group* nodefield = fields->createGroup("nodefield");
   nodefield->createViewString("association", "vertex");
   nodefield->createViewString("type", "scalar");
-  nodefield->createViewString("topology", "mesh");
+  nodefield->createViewString("topology", "cartesianesh");
```

Comment:
Looks like this has a typo:
```suggestion
  nodefield->createViewString("topology", "cartesian");
```

Are we calling `blueprint::verify` on this? 
* If not, can we please add it.
* If so, does it pass?